infrastructure/zk/src/database.ts

```diff
@@ -1,101 +1,201 @@
 import { Command } from 'commander';
 import * as utils from './utils';
 
-export async function reset() {
+export async function reset(opts: any) {
     await utils.confirmAction();
-    await wait();
-    await drop();
-    await setup();
+    await wait(opts);
+    await drop(opts);
+    await setup(opts);
 }
 
-export async function resetTest() {
-    process.env.DATABASE_URL = process.env.TEST_DATABASE_URL;
-    await utils.confirmAction();
+function getDals(opts: any): Map<string, string> {
+    let dals = new Map<string, string>();
+    if (!opts.prover && !opts.server) {
+        dals.set('core/lib/dal', process.env.DATABASE_URL!);
+        dals.set('prover/prover_dal', process.env.DATABASE_PROVER_URL!);
```

Comment:
nit: might it be better to put `core/lib/dal` and `prover/prover_dal` into the conts?